# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1303293345 -7200
#      Wed Apr 20 11:55:45 2011 +0200
# Node ID d65b8f5373b4617568e2c9f65d14ed068590a6f3
# Parent  3b7573c783cc1f9a522f59dc6130be9844ae38d0
Enlarged 100% scaling size, scaling is now displayed in percent.

diff --git a/gui/src/Plot2dWidget.cpp b/gui/src/Plot2dWidget.cpp
--- a/gui/src/Plot2dWidget.cpp
+++ b/gui/src/Plot2dWidget.cpp
@@ -30,17 +30,17 @@ void Plot2dView::initializeGL() {
     glEnable(GL_POLYGON_SMOOTH);
     glEnable(GL_BLEND);
     glBlendFunc(GL_SRC_ALPHA,GL_ONE_MINUS_SRC_ALPHA);
 }
 
 void Plot2dView::paintGL() {
     glMatrixMode(GL_MODELVIEW_MATRIX);
     glLoadIdentity();
-    glScaled(m_zoom, m_zoom, 0.0);
+    glScaled(m_zoom * 1.5, m_zoom * 1.5, 0.0);
     glTranslated(-0.5 - m_scrollX, -0.5 - m_scrollY, 0.0);
 
     glClear(GL_COLOR_BUFFER_BIT);
     glBegin(GL_LINES);
         glColor3d(1.0, 1.0, 1.0);
         glVertex2d(0.0, 0.0);
         glVertex2d(1.0, 0.0);
         glVertex2d(0.0, 0.0);
@@ -54,18 +54,18 @@ void Plot2dView::paintGL() {
                 glVertex2d(d, sin(d*2*3.141 + phi) / 2 + 0.5);
         glEnd();
     }
 
     glMatrixMode(GL_MODELVIEW_MATRIX);
     glLoadIdentity();
 
     glColor3d(1.0, 1.0, 1.0);
-    renderText(-0.9, -0.9, 0.0, QString("Scaling: %1, Translation: (%2/%3)")
-                                .arg(m_zoom)
+    renderText(-0.9, -0.9, 0.0, QString("Scaling: %1%, Translation: (%2/%3)")
+                                .arg(m_zoom * 100)
                                 .arg(m_scrollX)
                                 .arg(m_scrollY));
 }
 
 void Plot2dView::resizeGL(int w, int h) {
     glViewport(0, 0, w, h);
     glMatrixMode(GL_MODELVIEW_MATRIX);
     glLoadIdentity();
